models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -191,31 +191,34 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     pft.traits <- as.numeric(pft.traits)
     
     # Leaf carbon concentration
-    leafC <- 0.48  #0.5
+    leafC <- 0.48
     if ("leafC" %in% pft.names) {
       leafC <- pft.traits[which(pft.names == "leafC")]
       id <- which(param[, 1] == "cFracLeaf")
-      param[id, 2] <- leafC * 0.01  # convert to percentage from 0 to 1
+      param[id, 2] <- leafC * 0.01  # convert from percentage to fraction
     }
     
     # Specific leaf area converted to SLW
+    # leafCSpWt [gC/m2 leaf], SLA [m2 leaf/kg C], leafC [percentage C]
     SLA <- NA
     id <- which(param[, 1] == "leafCSpWt")
     if ("SLA" %in% pft.names) {
       SLA <- pft.traits[which(pft.names == "SLA")]
-      param[id, 2] <- 1000 * leafC * 0.01 / SLA
+      param[id, 2] <- 10 * leafC / SLA
```

Comment:
I could easily have gotten confused, but based on the units specified in the comments above, I _**think**_

Lets use "C" and "biomass" to make units more explicit:

- SLA [m2 leaf / kg _biomass_] 
- leafC [fraction C] == [kg C / kg biomass] 

(_note:_ comments say leaf C is `percentage C` but if leafC = 0.48, units are fraction C, as above). 

Then the conversion would be

```
                                          leafC [kg C / kg biomass]
leafCSpWt [gC/m2 leaf] = 1000 [g / kg] x ---------------------------
                                          SLA [m2 leaf / kg biomass]
```

and 

[updated: invert first term on RHS per [comment below](https://github.com/PecanProject/pecan/pull/3608#discussion_r2308175886)] 

```
                                                  leafC [kg C / kg biomass]
SLA [m2 leaf / kg biomass] = 1000 [g C / kg C ] x ---------------------------
                                                  leafCSpWt [g C / m2 leaf]
```

If that is the case, we can do:

```suggestion
      param[id, 2] <- PEcAn.utils::ud_convert(leafC / SLA, 'kg/m2', 'g/m2')
```

Reply:
The idea of specifying "biomass" and "C" is great! But I guess the scalar "0.001 kg/g" should be "1000g/kg" in your SLA calculation above to cancel off "kg C" and "g C" (if we could assume 1000 g C/kg C for 1000g/kg).

Reply:
I agreed that an official unit conversion would be better than just multiplying by 10.

Reply:
@Qianyuxuan good catch on the SLA calculation - I've updated it. 

Looks like I had the eqn on line 207 correct, but 209 incorrect (now corrected).